PresentScreenings/Utilities/ScreeningsPlanner.cs

```diff
@@ -115,6 +116,103 @@ public static string LogTimeString()
         #endregion
 
         #region Private Methods
+        private void AttendScreenings(string filmFan, List<Screening> screenings)
+        {
+            DateTime orgDay = _controller.Plan.CurrDay;
+            foreach (var screening in screenings)
+            {
+                bool fits = true;
+                if (screening is OnDemandScreening onDemandScreening)
+                {
+                    fits = FitOnDemandScreening(filmFan, onDemandScreening);
+                }
+                if (fits && screening.IsPlannable)
+                {
+                    screening.ToggleFilmFanAttendance(filmFan);
+                    screening.AutomaticallyPlanned = true;
+                    _controller.UpdateAttendanceStatus(screening);
+                    _plannedScreenings.Add(screening);
+                }
+                _controller.ReloadScreeningsView();
+            }
+            _controller.GoToDay(orgDay);
```

Comment:
Please write a use story like "As a dev I want automatic planning to minimize visual updates".

Reply:
Included it in #228 as task "Minimize visual updates in the daily schemes".